manageiq-schema.gemspec

```
@@ -25,10 +25,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency "more_core_extensions",    ">= 3.5", "< 5"
   spec.add_dependency "pg"
   spec.add_dependency "pg-pglogical",            "~> 2.1.1"
-  spec.add_dependency "rails",                   "~>6.0.0"
+  spec.add_dependency "rails",                   "~> 6.0.0", "> 6.0.6.1"
```

You missed the `=`  - this will fail without it.

```suggestion
  spec.add_dependency "rails",                   "~> 6.0.6", ">= 6.0.6.1"
```